react/README.md

```diff
@@ -74,56 +64,139 @@
 
 
 ## Declaration
-  - Do not use displayName for naming components. Instead, name the component by reference.
+  
+  - [3.1](#3.1) <a name='3.1'></a> **Display name**: If applying `displayName` to a component, it should match the name of the component itself.
+  
+  > NOTE: You can also use third-party plugins to programmatically add `displayName`, example with a [babel plugin](https://www.npmjs.com/package/babel-plugin-add-react-displayname)
 
   ```javascript
   // bad
   export default class ReservationCard extends Component {
-    displayName = 'ReservationCard';
+  }
+  
+  export default function ReservationCard() {
+    //
   }
 
   // good
   export default class ReservationCard extends Component {
+    static displayName = 'ReservationCard';
+  }
+  
+  export default function ReservationCard() {
+    //
   }
+  
+  ReservationCard.displayName = 'ReservationCard';
```

Comment:
if we are going to be using static for `propTypes`, `defaultProps`, and `contextTypes`, does it make sense to also use it for `displayName`?

Reply:
yup, ill update